lib/puppet/parser/functions/load_module_metadata.rb

```diff
@@ -17,14 +17,19 @@ module Puppet::Parser::Functions
   DOC
   ) do |args|
     raise(Puppet::ParseError, 'load_module_metadata(): Wrong number of arguments, expects one or two') unless [1, 2].include?(args.size)
+
     mod = args[0]
     allow_empty_metadata = args[1]
     module_path = function_get_module_path([mod])
     metadata_json = File.join(module_path, 'metadata.json')
 
     metadata_exists = File.exists?(metadata_json) # rubocop:disable Lint/DeprecatedClassMethods : Changing to .exist? breaks the code
     if metadata_exists
-      metadata = PSON.load(File.read(metadata_json))
+      metadata = if Puppet::Util::Package.versioncmp(Puppet.version, '8.0.0').negative?
```

Comment:
Rubocop prefers the usage of `.negative?` instead of `< 0`

Reply:
I believe the way versioncmp works is by returning `-1` if the value is lower, `0` is equal, and `1` if higher. Hence, the `.negative?` / `.positive?` methods. So on Puppet 8.0.0, it will return a `0`. I am unsure, however, if it returns a `0` or a `1` on Puppet 8.0.1+.

Reply:
Cool -  well we are running puppet 8.0.1 at the moment ([here](https://github.com/puppetlabs/puppetlabs-stdlib/actions/runs/5059183004/jobs/9080360953?pr=1307#step:4:328)) and looks to be all good so 👍 . I'll resolve this.